i18n/messages-ja.xtb

```diff
@@ -641,30 +641,30 @@
+	<translation id="4466763630197200161" key="MSG_BREADCRUMBS_ADMIN_LABEL" source="/usr/local/google/home/bryk/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="Label 'Admin' that appears as a breadcrumbs on the action bar.">管理者</translation>
+	<translation id="1459282068813879241" key="MSG_ADMIN_NAMESPACES_LABEL" source="/usr/local/google/home/bryk/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="Label that appears above the list of resources.">ネームスペース</translation>
+	<translation id="133343695009872751" key="MSG_ADMIN_NODES_LABEL" source="/usr/local/google/home/bryk/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="Label that appears above the list of resources.">ノード</translation>
+	<translation id="6870889542203892242" key="MSG_ADMIN_PERSISTENT_VOLUMES_LABEL" source="/usr/local/google/home/bryk/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="Label that appears above the list of resources.">永続ボリューム</translation>
+	<translation id="3195481899172694103" key="MSG_ACTION_BAR_CREATE_ACTION" source="/usr/local/google/home/bryk/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="Label for global action bar create button.">作成</translation>
+	<translation id="6165413233411120289" key="MSG_ACTION_BAR_CREATE_ACTION_TOOLTIP" source="/usr/local/google/home/bryk/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="Label for global action bar create button tooltip.">アプリケーションまたはKubernetesリソースを作成</translation>
+	<translation id="5207082267245498149" key="MSG_BREADCRUMBS_SERVICES_AND_DISCOVERY_LABEL" source="/usr/local/google/home/bryk/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="Label 'Services and discovery' that appears as a breadcrumbs on the action bar.">サービスとディスカバリー</translation>
+	<translation id="352791570684627676" key="MSG_SERVICES_AND_DISCOVERY_INGRESS_LABEL" source="/usr/local/google/home/bryk/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="Label that appears above the list of resources.">イングレス</translation>
+	<translation id="6306881076646050049" key="MSG_SERVICES_AND_DISCOVERY_SERVICES_LABEL" source="/usr/local/google/home/bryk/src/github.com/kubernetes/dashboard/.tmp/serve/app-dev.js" desc="Label that appears above the list of resources.">サービス</translation>
 	<translation id="173829199104422763" key="MSG_LOGS_TITLE_FROM" source="/home/maciaszczykm/workspace/dashboard/.tmp/serve/app-dev.js" desc="Title prefix for logs card.">Logs from</translation>
 	<translation id="545013982201122635" key="MSG_LOGS_TITLE_IN" source="/home/maciaszczykm/workspace/dashboard/.tmp/serve/app-dev.js" desc="Title part for logs card.">in</translation>
 	<translation id="9015703491006359663" key="MSG_LOGS_TITLE_TO" source="/home/maciaszczykm/workspace/dashboard/.tmp/serve/app-dev.js" desc="Footer part for logs card.">to</translation>
```

Comment:
There is one problem.
These 3 lines (l.653-655) is too hard to translate to Japanese.
Because, English and Japanese sentence is sometimes reversed.
e.g.) in this case, "logs from `A` in `B`" will be translated to "`B` 内の `A` からのログ".

So in order to translate these, we have to change to sentence. Like [this line](https://github.com/kubernetes/dashboard/blob/27da4313cb22898c2134328665524bfadd8409a5/i18n/messages-en.xtb#L24).